### PR TITLE
Hello world.

### PR DESCRIPTION
**Pull request by joeadams04**

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
```
Comment (rjw57) [inline src/components/Main.js]: ``Component`` is never used here, yes? Maybe just
```jsx
import React from 'react';
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "iar-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-scripts": "1.1.0"
+  },
```
Comment (rjw57) [inline package.json]: Add 'material-ui'? Probably via ``npm install --save material-ui``.

File: src/components/Main.js (hunk 1 above)
Comment (joeadams04) [reply src/components/Main.js]: Correct. Component isn't used at the moment. This was me purely testing a functional component. Will remove Component for now until we start UI and refactor the existing components.

**Comment by rjw57** [comment]
Tested locally.

**Pull request merged**